### PR TITLE
Fixed broken formula solid.rb

### PR DESCRIPTION
**Pull request by tzwenn**
Patched two small bugs in code, that prevented solid from building successfully on clang.
Also updated style on formula, and fixed issues pointed out by `brew audit --strict` (except missing test).

### CODE REVIEW

File: Library/Formula/solid.rb [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+
+  deprecated_option "enable-doubles" => "with-doubles"
+  deprecated_option "enable-tracer" => "with-tracer"
 
-  option 'enable-doubles', 'Use internal double precision floats'
-  option 'enable-tracer', 'Use rounding error tracer'
+  option "with-doubles", "Use internal double precision floats"
+  option "with-tracer", "Use rounding error tracer"
+
+  patch :DATA
```
Comment (mistydemeo) [inline Library/Formula/solid.rb]: What's the source of this patch? has it been reported upstream?

**Comment by tzwenn** [comment]
Got it by manually with  `brew install --interactive --git` when fixing the failing build with clang-600.0.56.
Wrote a bug report to the developers by mail, no answer so far.

However their last version 3.5.7 (same issues, dates to 2013-06-18) works with gcc (tested 4.6 on Ubuntu), so I'd guessed this is not their primary concern.

**Comment by mistydemeo** [comment]
Can you add a comment above the patch block with a link to the mailing list post where you reported it, please?

**Comment by tzwenn** [comment]
(done, okay this way?)

**Comment by mistydemeo** [comment]
Thanks! Usually we prefer the comment above `patch :DATA`. Is there a web interface for this software where you could get a link to your bug report?

**Comment by tzwenn** [comment]
No, unfortunately there is not. All you find on their site is an email address: http://www.dtecta.com/support/ (you have to visit the main site to find a (flash!)-navigator).

File: Library/Formula/solid.rb [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+  sha1 "bd0afef7842f826c270cff32fc23994aec0b0d65"
+
+  deprecated_option "enable-doubles" => "with-doubles"
+  deprecated_option "enable-tracer" => "with-tracer"
 
-  option 'enable-doubles', 'Use internal double precision floats'
-  option 'enable-tracer', 'Use rounding error tracer'
+  option "with-doubles", "Use internal double precision floats"
+  option "with-tracer", "Use rounding error tracer"
+
```
Comment (mikemcquaid) [inline Library/Formula/solid.rb]: Trailing whitespace on this line (see `brew audit`)

**Pull request closed without merging**